### PR TITLE
Bug/add fivetran

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

## 🔧 Bug Fixes
- Updated models `stg_pendo__feature_event` and `stg_pendo__page_event` to include `_fivetran_id`, which was recently added per the Fivetran Pendo Connector's [release notes](https://fivetran.com/docs/applications/pendo/changelog#december2022). ([#19](https://github.com/fivetran/dbt_pendo_source/pull/19))

## ✨ Features
- Update documentation for `_fivetran_id`. ([#19](https://github.com/fivetran/dbt_pendo_source/pull/19))
- Revised source yaml and readme instructions for setting up the `GROUP` table with Snowflake. ([#19](https://github.com/fivetran/dbt_pendo_source/pull/19))

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [x] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)
Added `_fivetran_id`.

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature #18 
- [ ] No 

**How did you test the PR changes?** 

 

- [x] Buildkite  
- [x] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
♦️

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Comment by fivetran-catfritz** [comment]
To expand on this, `_fivetran_id` was added in [this connector update](https://fivetran.com/docs/applications/pendo/changelog#december2022) to tables `PAGE_EVENT` and `FEATURE_EVENT`. the and was added to the columns indicated as unique keys in the ERD. I added _fivetran_id to the surrogate keys and its own field in the staging tables since they were the combination of the unique keys. 

For some extra background, the _fivetran_id for PAGE_EVENT is hashed on the and `param_*` fields and
FEATURE_EVENT hashed on the `property_*` and `param_*` fields.

**Review by fivetran-sheringuyen — APPROVED** [review]
Hey @fivetran-catfritz, I left a few comments but nothing that should block this release. Approved!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# dbt_pendo_source v0.4.0
+
+## 🚨 Breaking Changes 🚨:
+## 🔧 Bug Fixes
+- Updated models `stg_pendo__feature_event` and `stg_pendo__page_event` to include `_fivetran_id`, which was recently added per the Fivetran Pendo Connector's [release notes](https://fivetran.com/docs/applications/pendo/changelog#december2022). ([#19](https://github.com/fivetran/dbt_pendo_source/pull/19))
+
+## ✨ Features
+- Update documentation for `_fivetran_id`. ([#19](https://github.com/fivetran/dbt_pendo_source/pull/19))
+- Revised source yaml and readme instructions for setting up the `GROUP` table with Snowflake. ([#19](https://github.com/fivetran/dbt_pendo_source/pull/19))
```
[the remark this answers is not in the document]
Comment (fivetran-catfritz) [reply CHANGELOG.md]: Done!

**Review by fivetran-sheringuyen — APPROVED** [review]

File: .buildkite/hooks/pre-command [hunk 2]
```diff
@@ -21,4 +21,5 @@ export CI_SNOWFLAKE_DBT_USER=$(gcloud secrets versions access latest --secret="C
 export CI_SNOWFLAKE_DBT_WAREHOUSE=$(gcloud secrets versions access latest --secret="CI_SNOWFLAKE_DBT_WAREHOUSE" --project="dbt-package-testing-363917")
 export CI_DATABRICKS_DBT_HOST=$(gcloud secrets versions access latest --secret="CI_DATABRICKS_DBT_HOST" --project="dbt-package-testing-363917")
 export CI_DATABRICKS_DBT_HTTP_PATH=$(gcloud secrets versions access latest --secret="CI_DATABRICKS_DBT_HTTP_PATH" --project="dbt-package-testing-363917")
-export CI_DATABRICKS_DBT_TOKEN=$(gcloud secrets versions access latest --secret="CI_DATABRICKS_DBT_TOKEN" --project="dbt-package-testing-363917")
\ No newline at end of file
+export CI_DATABRICKS_DBT_TOKEN=$(gcloud secrets versions access latest --secret="CI_DATABRICKS_DBT_TOKEN" --project="dbt-package-testing-363917")
+export CI_DATABRICKS_DBT_TOKEN=$(gcloud secrets versions access latest --secret="CI_DATABRICKS_DBT_CATALOG" --project="dbt-package-testing-363917")
```
Comment (fivetran-sheringuyen) [inline .buildkite/hooks/pre-command:25]: you forgot to update the variable

**Pull request merged**